openapi_server/apis/users_api.py

```diff
@@ -39,8 +40,19 @@
 async def create_user(
     create_user: CreateUser = Body(None, description="Created user object"),
 ) -> User:
-    """This can only be done by the logged in user."""
-    ...
+    """"""
+    if not DbUser.exists():
+        DbUser.create_table(read_capacity_units=1, write_capacity_units=1, wait=True)
```

Comment:
Let's put this aside from the api handlers. say dynamodb_setup.py and call it from app.py. And the dynamodb_setup.py do all table check for the models and also create indexes if not exists.